src/internal/connector/onedrive/restore.go

```diff
@@ -65,18 +68,20 @@ func RestoreCollections(
 
 	// Iterate through the data collections and restore the contents of each
 	for _, dc := range dcs {
-		parentPerms, ok := parentPermissions[dc.FullPath().String()]
-		if !ok {
-			onedrivePath, err := path.ToOneDrivePath(dc.FullPath())
-			if err != nil {
-				errUpdater(dc.FullPath().String(), fmt.Errorf("invalid restore path"))
-			}
+		if opts.RestorePermissions {
+			parentPerms, ok = parentPermissions[dc.FullPath().String()]
```

Comment:
parentPerms isn't getting used anywhere that I can see.  
```suggestion
			_, ok = parentPermissions[dc.FullPath().String()]
```

Reply:
It gets used as an arg to RestoreCollections
https://github.com/alcionai/corso/blob/8c91de1ca2be61145cd7c73cd66f62c8fae5642f/src/internal/connector/onedrive/restore.go#L92